index.js

```diff
@@ -62,7 +62,8 @@ var Cordova = function(id, emitter, args, logger, config) {
           return;
         }
         var newUrl = url + "?id=" + id;
-        newUrl = newUrl.replace(/localhost/g, "10.0.2.2");
+        var ip = self.settings.hostip;
```

Comment:
Can we add a default IP, in case the developer doesn't specify one? Also add documentation in the README to that effect?